blockstack/blockstack-browser
# Feature/association tokens

This branch adds support for generating and passing association tokens to the auth response on user sign-in.  This enables the user to authenticate and leverage a private Gaia hub.

To test, you can do the following:

* Set up a Gaia hub.  I was able to run one on `localhost` for example.
   * Be sure to whitelist your ID-addresses in order to make it private!
* Set the Gaia Hub URL in the browser's API settings to the write endpoint of your Gaia hub
* Sign in to a (modern) Blockstack app.  Everything should "just work"
   * You can directly confirm that the `associationToken` is present in the Gaia hub config by inspecting `localStorage['blockstack-gaia-hub-config']`.
